controllers/vspheremachine_controller.go

```diff
@@ -350,7 +364,7 @@ func (r machineReconciler) reconcileNormal(ctx *context.MachineContext) (reconci
 				"unexpected error while reconciling network for %s", ctx)
 		}
 		ctx.Logger.Info("network is not reconciled")
-		return reconcile.Result{}, nil
+		return reconcile.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
This is a hack and should be avoided long-term. Please add a `// TODO` to indicate this should be revisited.